src/FunctionLoader.cs

```diff
@@ -7,7 +7,7 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-
+using System.Management.Automation;
```

Comment:
Is this using directive still needed?

Reply:
Yes this is needed to access Platform.IsWindows, as I need to only add this for Windows

Reply:
Ah, yes, you are right.